CMakeLists.txt

```diff
@@ -117,6 +117,7 @@ if (BUILD_SDF)
   ########################################
   # Find ignition utils
   ign_find_package(ignition-utils1 VERSION REQUIRED)
+  ign_find_package(ignition-utils1 COMPONENTS cli REQUIRED_BY usd)
```

Comment:
Can these calls be combined?

Reply:
I think they need to be in separate calls if there are different keywords. Arguably the `cli` component could be `PRIVATE` since it is just used to build executables, but we haven't tried to mark that here. So I think it would be fine to combine these two lines into:

~~~
ign_find_package(ignition-utils1 REQUIRED COMPONENTS cli)
~~~

Reply:
fixed in https://github.com/ignitionrobotics/sdformat/pull/966/commits/5600ab6b92dd9d7ee322f1d64725ccb8a5c41358